src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2870,13 +2872,19 @@ def aks_update(cmd, client, resource_group_name, name,
                        '"--enable-azure-rbac" or '
                        '"--disable-azure-rbac" or '
                        '"--enable-public-fqdn" or '
-                       '"--disable-public-fqdn"')
+                       '"--disable-public-fqdn" or '
+                       '"--tags"')
 
     if not enable_managed_identity and assign_identity:
         raise CLIError(
             '--assign-identity can only be specified when --enable-managed-identity is specified')
 
     instance = client.get(resource_group_name, name)
+
+    # update tags
+    if tags is not None:
```

Comment:
I think we may need to merge the tags with the current one.

Reply:
Confirming the expected behavior of this option with PM.

For now, if the user provides an empty string when specifying the `--tags` option, the original tags will be cleared. If the user does not explicitly specify the `--tags` option, the original tags will not be changed.

Reply:
Confirmed with PM, the replacement update is the expected behavior.